traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -608,43 +580,15 @@ func (ds *TODeliveryService) Read(h http.Header, useIMS bool) ([]interface{}, er
 		// NOTE: it's required to handle minor version cases in a descending >= manner
 		case version.Major > 3:
 			returnable = append(returnable, ds.RemoveLD1AndLD2())
-		case version.Major > 2 && version.Minor >= 1:
+		case version.Major == 3 && version.Minor >= 1:
 			returnable = append(returnable, ds.DowngradeToV31())
-		case version.Major > 2:
-			returnable = append(returnable, ds.DowngradeToV31().DeliveryServiceV30)
```

Comment:
I think this part may be important for differentiating v3.0 and v3.1

Reply:
Oops! Re-added but with a `3` in 9e4ad30373.